**(1)** Find $$\lim_{x\rightarrow{}0^{+}}\sin{}x\ln{}x$$

Make sure to check that you only use LHR when there is an indeterminate form.

$$\lim_{x\rightarrow{}0^{+}}\sin{}x\ln{}x=\lim_{x\rightarrow{}0^{+}}\frac{\ln{}x}{\csc{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0^{+}}\frac{\frac{1}{x}}{-\csc{}x\cdot{}\cot{}x}=$$

$$\lim_{x\rightarrow{}0^{+}}\frac{-\sin^{2}x}{x\cos{}x}=\frac{0}{0}$$

Use LHR again for $$\lim_{x\rightarrow{}0^{+}}\frac{-\sin^{2}x}{x\cos{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0^{+}}\frac{-2\sin{}x\cos{}x}{\cos{}x-x\sin{}x}=\frac{0}{1-0}=0$$

**(2)** Find $$\lim_{x\rightarrow{}0}\frac{e^{x}-e^{-x}-2x}{x-\sin{}x}$$

Make sure to check that you only use LHR when there is an indeterminate form.

$$\lim_{x\rightarrow{}0}\frac{e^{x}-e^{-x}-2x}{x-\sin{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0}\frac{e^{x}+e^{-x}-2}{1-\cos{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0}\frac{e^{x}-e^{-x}}{\sin{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0}\frac{e^{x}+e^{-x}}{\cos{}x}=\frac{1+1}{1}=2$$

**(3)** A cylindrical can is to be made to hold one litre of oil. Find the dimensions that will minimize the cost of the metal to manufacture the can.

**Step 1:** Find a function for that which you are trying to optimize, in this case the area of the can. $$A(r)=2r^{2}\pi+2\pi{}rh$$

**Step 2:** Use the side constraint $$1=r^{2}\pi{}h$$ to replace $h$ in terms of $r$ $$A(r)=2r^{2}\pi+\frac{2\pi{}r}{r^{2}\pi}=A(r)=2\left(r^{2}\pi+\frac{1}{r}\right)$$

**Step 3:** Differentiate and set to zero. $$A'(r)=2\left(2r\pi-\frac{1}{r^{2}}\right)=0$$

Solve the equation for $$r=\sqrt[3]{\frac{1}{2\pi}}=0.54193$$

Finally, use the side constraint for $$h=\frac{1}{\left(\frac{1}{2\pi}\right)^{\frac{2}{3}}\pi}=1.0839$$ **Answer:** The dimensions that will minimize the cost of the metal to manufacture the can are $r=0.54193$dm and $h=1.0839$dm.

**(4)** A farmer has 2400ft of fencing and wants to fence off a rectangular field that borders a straight river. She needs no fence along the river. What are the dimensions of the field that has the largest area?

**Step 1:** $$A(w)=l\cdot{}w$$

**Step 2:** $l+2w=2400$, therefore $l=2400-2w$ and $$A(w)=(2400-2w)\cdot{}w=2400w-2w^{2}$$

**Step 3:** Differentiate and set to zero. $$A'(w)=2400-4w=0$$ **Answer:** The dimensions of the field that has the largest area are $w=600,l=1200$.

**(5)** Find $$\lim_{x\rightarrow{}1}\frac{1-x+\ln{}x}{1+\cos\pi{}x}$$

Make sure to check that you only use LHR when there is an indeterminate form.

$$\lim_{x\rightarrow{}1}\frac{1-x+\ln{}x}{1+\cos\pi{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}1}\frac{-1+\frac{1}{x}}{-\pi\sin\pi{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}1}\frac{-\frac{1}{x^{2}}}{-\pi^{2}\cos\pi{}x}=-\frac{1}{\pi^{2}}$$

**(6)** How close does the semi-circle $y=\sqrt{16-x^{2}}$ come to the point $P=(1,\sqrt{3})$?

<img src="optsemi.png">

**Step 1:** Define $$f(a)=\sqrt{a^{2}+b^{2}}$$

**Step 2:** Use the fact that $Q$ is on the semi-circle to replace $b$ in terms of $a$ $$b+\sqrt{3}=\sqrt{16-(a+1)^{2}}$$

$$f(a)=\sqrt{a^{2}+\left(\sqrt{16-(a+1)^{2}}-\sqrt{3}\right)^{2}}$$

**Step 3:** Differentiate and set to zero $$f'(a)=\frac{-2-2\sqrt{3}\frac{-2a-2}{2\cdot\sqrt{-a^{2}-2a+15}}}{2\cdot\sqrt{18-2a-2\sqrt{3}\sqrt{-a^{2}-2a+15}}}=0$$

Recall that a fraction is zero if and only if the numerator is zero. The equation simplifies to $$a^{2}+2a-3=0$$ for which the solutions are $a=-3,a=1$. **Answer:** The coordinates of $Q$ are $(2,\sqrt{12}-\sqrt{3})$ and the distance from $P$ to $Q$ is 2 (think about why this is not surprising). Note that the solution $a=-3$ provides us with the maximum distance of a point on the semi-circle to $P$.

**(7a)**

In [10]:
x=var('x')

In [2]:
f(x)=x^6-x^5-6*x^4-x^2+x+10

In [3]:
fp(x)=diff(f(x),x)

$$f'(x)=6 \, x^{5} - 5 \, x^{4} - 24 \, x^{3} - 2 \, x + 1$$

Since $f(1)=4$ and $f(2)=-56$, I know that there is a $x$-intercept in the interval $[1,2]$. I choose $x_{1}=1$.

In [24]:
x1=1

In [25]:
x2=x1-(f(x1)/fp(x1))

In [26]:
n(x2)

1.16666666666667

In [27]:
x3=x2-(f(x2)/fp(x2))

In [28]:
n(x3)

1.14008652743594

In [29]:
x4=x3-(f(x3)/fp(x3))

In [30]:
n(x4)

1.13929444014290

In [31]:
x5=x4-(f(x4)/fp(x4))

In [32]:
n(x5)

1.13929374596799

One solution for the equation $x^{6}-x^{5}-6x^{4}-x^{2}+x+10=0$ is $x\approx{}1.39294$.

**(7b)**

In [33]:
f(x)=(4/(x^2+1))-x^2*(4-x^2)

In [34]:
fp(x)=diff(f(x),x)

$$f'(x)=2 \, x^{3} + 2 \, {\left(x^{2} - 4\right)} x - \frac{8 \, x}{{\left(x^{2} + 1\right)}^{2}}$$

Since $f(0)=4$ and $f(1)=-1$, there must be an $x$-intercept in the interval $[0,1]$. I choose $x_{1}=1$ again.

In [39]:
x1=1

In [40]:
x2=x1-(f(x1)/fp(x1))

In [41]:
n(x2)

0.833333333333333

In [42]:
x3=x2-(f(x2)/fp(x2))

In [43]:
n(x3)

0.843092535766353

In [44]:
x4=x3-(f(x3)/fp(x3))

In [45]:
n(x4)

0.843108209827693

In [46]:
x5=x4-(f(x4)/fp(x4))

In [47]:
n(x5)

0.843108209870886

One solution for the equation $x^{2}(4-x^{2})=\frac{4}{x^{2}+1}$ is $x\approx{}0.84311$.

**(7c)**

In [48]:
f(x)=1-x^2*sqrt(2-x-x^2)

In [49]:
fp(x)=diff(f(x),x)

$$f'(x)=\frac{{\left(2 \, x + 1\right)} x^{2}}{2 \, \sqrt{-x^{2} - x + 2}} - 2 \, \sqrt{-x^{2} - x + 2} x$$

Since $f(-1)=1-\sqrt{2}$ and $f(0)=1$, there must be an $x$-intercept in the interval $[-1,0]$. $f$ has a critical point at $x=0$, so I choose $x_{1}=-1$ instead.

In [59]:
x1=-1

In [60]:
x2=x1-(f(x1)/fp(x1))

In [61]:
n(x2)

-0.832632446392313

In [62]:
x3=x2-(f(x2)/fp(x2))

In [63]:
n(x3)

-0.826476364377094

In [64]:
x4=x3-(f(x3)/fp(x3))

In [65]:
n(x4)

-0.826462326269352

In [66]:
x5=x4-(f(x4)/fp(x4))

In [67]:
n(x5)

-0.826462326195434

One solution for the equation $x^{2}\sqrt{2-x-x^{2}}=1$ is $x\approx{}-0.82646$.

**(7d)**

In [68]:
f(x)=x^2-x+1-4*exp(-x^2)*sin(x)

In [70]:
fp(x)=diff(f(x),x)

$$f'(x)=8 \, x e^{\left(-x^{2}\right)} \sin\left(x\right) - 4 \, \cos\left(x\right) e^{\left(-x^{2}\right)} + 2 \, x - 1$$

Since $f(0)=1$ and $f(1)<0$, there must be an $x$-intercept in the interval $[0,1]$. I choose $x_{1}=0$.

In [76]:
x1=0

In [77]:
x2=x1-(f(x1)/fp(x1))

In [78]:
n(x2)

0.200000000000000

In [79]:
x3=x2-(f(x2)/fp(x2))

In [80]:
n(x3)

0.218832730610920

In [81]:
x4=x3-(f(x3)/fp(x3))

In [82]:
n(x4)

0.219163571329239

In [83]:
x5=x4-(f(x4)/fp(x4))

In [84]:
n(x5)

0.219163677236341

One solution for the equation $4e^{-x^{2}}\sin{}x=x^{2}-x+1$ is $x\approx{}0.21916$.

**(7e)**

In [85]:
f(x)=3*sin(x^2)-2*x

In [86]:
fp(x)=diff(f(x),x)

$$f'(x)=6 \, x \cos\left(x^{2}\right) - 2$$

I was lucky. I plugged in $x=0$ and there is a solution for the equation. I no longer need Newton's method.

**(8)** For a fish swimming at a speed $v$ relative to the water, the energy expenditure per unit time is proportional to $v^{3}$. It is believed that migrating fish try to minimize the total energy required to swim a fixed distance. If the fish are swimming against a current $u$ ($u<v$), then the time required to swim a distance $L$ is $L/(v-u)$ and the total energy $E$ required to swim the distance is given by $$E(v)=av^{3}\cdot\frac{L}{v-u}$$ where $a$ is the proportionality constant. Determine the value of $v$ that minimizes $E$.

Differentiate $$E'(v)=\frac{Lav^{2}}{v-u}\left(3-\frac{v}{(v-u)}\right)$$

Set the derivative to zero for the answer. **Answer:** $v$ is minimized at $v=\frac{3}{2}u$.

**(9)** A cone with height $H$ and radius $R$ is inscribed in a larger cone with height $h$ and radius $r$ so that its vertex is at the centre of the base of the larger cone. Find $R$ in terms of the dimensions of the larger cone that makes the volume of the smaller cone maximal.

**Step 1:** $$V(R)=\frac{1}{3}R^{2}\pi{}H$$

**Step 2:** $$\frac{h-H}{R}=\frac{h}{r}$$ and therefore $$H=\frac{h}{r}(r-R)$$

**Step 3:** Differentiate and set to zero $$V(R)=\frac{1}{3}R^{2}\pi\frac{h}{r}(r-R)$$

$$V'(R)=\pi{}hR\left(\frac{2}{3}-\frac{R}{r}\right)=0$$

**Answer:** There are two solutions here, $R=0$ for the minimum and $R=\frac{2}{3}r$ for the maximum.

**(10)** Find $$\lim_{x\rightarrow{}1}\left(\frac{x}{x-1}-\frac{1}{\ln{}x}\right)$$

Make sure to check that you only use LHR when there is an indeterminate form.

$$\lim_{x\rightarrow{}1}\left(\frac{x}{x-1}-\frac{1}{\ln{}x}\right)=\lim_{x\rightarrow{}1}\frac{x\ln{}x-(x-1)}{(x-1)\ln{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}1}\frac{x\ln{}x}{x\ln{}x+(x-1)}=$$

$$\left(1+\lim_{x\rightarrow{}1}\frac{x-1}{x\ln{}x}\right)^{-1}\stackrel{\mbox{LHR}}{=}\left(1+\lim_{x\rightarrow{}1}\frac{1}{\ln{}x+1}\right)^{-1}=\left(1+\frac{1}{1}\right)^{-1}=\frac{1}{2}$$

**(11)** Of the infinitely many lines that are tangent to the curve $$y=-\sin{}x$$ and pass through the origin, there is one that has the largest slope. Use Newton's Method to find the slope of that line.

<img src="optsine.png">

The slope of the tangent line is $$S(x)=-\frac{\sin{}x}{x}$$ Differentiate and set to zero $$S'(x)=\frac{\sin{}x-x\cos{}x}{x^{2}}=0$$

In [91]:
f(x)=sin(x)-x*cos(x)

In [101]:
find_root(f,pi,2*pi)

4.493409457909064

Instead of Newton's method (which is simple enough to use here), I have used a function in SageMath to approximate the solution. **Answer:** the slope of the line is approximately $0.21723$.

<img src="optparabola1.png">

<img src="optparabola2.png">

<img src="optparabola3.png">

**(13)** Find the area of the largest rectangle that can be inscribed in a semicircle of radius $r$.

**Step 1:** The function to optimize is $A(l)=l\cdot{}w$.

**Step 2:**: The side constraint $r^{2}=w^{2}+\frac{l^{2}}{4}$ gives us $$A(l)=l\cdot\sqrt{r^{2}-\frac{l^{2}}{4}}$$

**Step 3:** Differentiate and set to zero. $$A'(l)=\sqrt{r^{2}-\frac{l^{2}}{4}}-\frac{l^{2}}{4\sqrt{r^{2}-\frac{l^{2}}{4}}}=0$$

Solve this equation for $l=\sqrt{2}r,w=\frac{1}{\sqrt{2}}r$. **Answer:** The area of the largest rectangle that can be
inscribed in a semicircle of radius $r$ is $r^{2}$.